lemur/plugins/lemur_aws/tests/test_plugin.py

```diff
@@ -167,13 +168,35 @@ def test_get_all_elb_and_elbv2s(app, aws_credentials):
     set_plugin_option("accountNumber", "123456789012", options)
     set_plugin_option("endpointType", "elb", options)
     set_plugin_option("regions", "us-east-1", options)
-    elbs = aws_source.get_endpoints(options)
+
+    # Note: required because mocking describe_listener_certificates API call not yet supported by moto.
+    with mock.patch.object(
```

Comment:
`moto` doesn't properly support mocking secondary certificates (as per comment) but also wanted to note here a similar approach can be used for mocking out other simple cloud provider API calls in tests that may be applicable for testing some of the planned destination plugins.